crates/core/src/protocols/evalcheck/prove.rs

```diff
@@ -60,9 +60,7 @@ where
 	// Internally used to collect subclaims without evaluations for future query and memoization
 	claims_without_evals: Vec<(MultilinearPolyOracle<F>, EvalPoint<F>)>,
 	// The list of claims that reduces to a bivariate sumcheck in a round.
-	projected_bivariate_claims: Vec<EvalcheckMultilinearClaim<F>>,
-	// The list of claims that reduces to a composite MLEcheck in a round.
-	composite_mle_claims: Vec<EvalcheckMultilinearClaim<F>>,
+	bivariate_claims: Vec<BivariateClaims<P::Scalar>>,
```

Comment:
"bivariate" is incorrectly named. The `Composite` sumcheck claims are not bivariate products compositions. You can just name this field `sumcheck_claims`.